vdso/makefile

```diff
@@ -1,21 +1,25 @@
+arch = $(shell uname -m)
+
 # Specify extensions of files to delete when cleaning
 CLEANEXTS   = o so
 
 # Specify the source files, the target files,
 SOURCES     = vdso.cc vdso_time.cc
 OUTPUTFILE  = vdso.so
 
+LINKER_SCRIPT = vdso_$(arch).lds
+
 .PHONY: all
 all: $(OUTPUTFILE)
 
 $(OUTPUTFILE): $(subst .cc,.o,$(SOURCES))
 	$(CC) $(CCFLAGS) -I. \
 	-O2 -std=c++11 \
-	-fPIC -fno-stack-protector -fuse-ld=gold -m64 -shared -nostdlib \
+	-fPIC -fno-stack-protector -fuse-ld=gold -shared -nostdlib \
```

Comment:
is there a reason to drop the -m64 flag?

Reply:
"-m64" seems to not be a valid option for Aarch64 according to the [GCC - index](https://gcc.gnu.org/onlinedocs/gcc/Option-Index.html).